samples/simple.js

```diff
@@ -4,12 +4,12 @@ const path = require('path');
 
 const nconf = require('nconf');
 
-require('./../lib/nconf_dir');
+require('./../lib/nconfetti');
```

Comment:
what do you think is better - this or `require('./../index');` (to use the real package export)